app/controllers/auth/registrations_controller.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+module Auth
+  class RegistrationsController < Devise::RegistrationsController
+    # The default destroy is changed to only deactivate the user, based on the official devise github
+    def destroy
+      resource.update(deactivated_at: DateTime.now) if resource.deactivated_at.present?
+      sign_out
+      set_flash_message! :noticed, :destroyed
+      respond_with_navigational(resource) { redirect_to after_sign_out_path_for(resource_name) }
```

Comment:
Gondolom ez az eredeti devise-os kontrollerből jön. Nekünk szerintem nem kell ennyire absztrakt kódot használni.
pl.: https://stackoverflow.com/questions/6770193/how-does-respond-with-navigational-work Ő visszafejti, hogy az első `respond_with_navigational(resource)` mi csinál, but I am not him. :(
Itt inkább használjunk egy `redirect_to sign_in_path` vagy valami hasonlót. Irányítsuk át úgy a felhasználót, hogy ne kapjon hibakódot, és egy olyan oldalra kerüljön, amit láthat.